### DISCUSSION TITLE
Where can I read the test log

### DISCUSSION THREAD

**Discussion by yuyenews** [TechEmpower/FrameworkBenchmarks#6403] [General]
Yesterday I checked my test results and found that Best performance was only 1, and Errors were 324,334. This is very different from my local self-test results, which is a bit strange.

So I want to look at the log to find problems and optimize. I don’t know where to view the log. The name of the framework is martian, which is a java framework.

Thank you

**Comment by joanhey** [comment]
https://tfb-status.techempower.com/unzip/results.2021-02-26-17-50-16-534.zip/results/20210222011946/martian

**Comment by joanhey** [comment]
https://tfb-status.techempower.com/unzip/results.2021-02-26-17-50-16-534.zip/results/20210222011946/martian/run/martian.log

**Reply by bdw429s** [reply]
@joanhey What is the pattern to guess these URLs?  I'd also like to see the logs for the benchmark run that started at `2021-02-26 at 5:54 PM` but I'm not sure what the pattern is for the timestamps in the URL.  And if you back the URL up to https://tfb-status.techempower.com/unzip/ then you get a 404.

**Comment by joanhey** [comment]
https://tfb-status.techempower.com/

When the run finish, you can watch the logs.
1)
![Captura de pantalla_área-de-selección_20210301193300](https://user-images.githubusercontent.com/249085/109542396-49309a00-7ac5-11eb-927b-ab565f02e97d.png)
2)
![Captura de pantalla_área-de-selección_20210301193335](https://user-images.githubusercontent.com/249085/109542423-53eb2f00-7ac5-11eb-838a-d90bc59dbb45.png)
If your framework failed, you can click directly in the name
3)
![Captura de pantalla_área-de-selección_20210301193742](https://user-images.githubusercontent.com/249085/109542648-a3315f80-7ac5-11eb-959b-5b802815441e.png)
4)
![Captura de pantalla_área-de-selección_20210301193411](https://user-images.githubusercontent.com/249085/109542466-61081e00-7ac5-11eb-8a21-4974c95c7262.png)
5)
![Captura de pantalla_área-de-selección_20210301193430](https://user-images.githubusercontent.com/249085/109542493-69605900-7ac5-11eb-86e3-69819adf7dae.png)

**Reply by bdw429s** [reply]
@joanhey Thanks for showing me where that is found!

From what I gather, this means I won't be able to see what happened in my `coldbox-adobe` framework for the current run for another ~50 hours when it completes :disappointed:
